src/test/resources/resource_response.json

```diff
@@ -0,0 +1,41 @@
+{
+  "Items": [
```

Comment:
This makes me wonder, is items actually ever > 1?

Reply:
Yes, we get one item per created or modified version.